MicroQRArt/ViewModel/MyQRCodeListViewModel.swift

```diff
@@ -112,4 +114,8 @@ extension MyQRCodeListViewModel: MyQRCodeListViewModelOutputs {
     var errorMessage: Signal<String, Never> {
         return _errorMessage
     }
+    
+    var isLoading: Property<Bool> {
+        return _isLoading
+    }
```

Comment:
@donuts-yoshidahikaru 
ここは直接model.isLoadingそのまま使えそうです。
```suggestion
    var isLoading: Property<Bool> {
        return model.isLoading
    }
```

Reply:
@donuts-wuchunyen 

8a2206b409c03997b32cb42cfd74e2721b1b026f で修正いたしました。

`errorMessage` も同様に修正いたしました。

```diff
-    var errorMessage: Signal<String, Never> {
-        return = _errorMessage
-    }
+    var errorMessage: Signal<String, Never> {
+        return model.errorMessage
+    }
```